crates/diesel_models/src/query/payment_attempt.rs

```diff
@@ -171,11 +171,23 @@ impl PaymentAttempt {
         merchant_id: &common_utils::id_type::MerchantId,
         connector_txn_id: &str,
     ) -> StorageResult<Self> {
+        let (txn_id, txn_data) = common_utils::types::ConnectorTransactionId::form_id_and_data(
```

Comment:
cc: @Narayanbhat166 can you pls review this?

Reply:
I will be reverting this for now, can be added and discussed later.

Why?

This was added for Worldpay, and they send back transactionReference (`payment_attempt_id` for hyperswitch), so this query will not be invoked